registry/server/core/apps/services/reduceApps.ts

```diff
@@ -0,0 +1,43 @@
+import _ from 'lodash/fp';
```

Comment:
- Make it generic
- Pick more specific name
- Remove 1-2 line functions that you use only once

Reply:
I created `postProcessResponse` for db
So it gives opportunities to process result of queries before they will be returned from db queries